index.html

```diff
@@ -167,7 +167,7 @@ <h2 class="meetup-date">24 nov</h2>
     .then((response) => {
       response.json().then((data) => {
         let jobs = data.map(d => {
-          let splitTag = d.title.includes(' na ') ? ' na ' : ' at '
+          let splitTag = d.title.includes(' na ') ? ' na ' : d.title.includes(' no ') ? ' no ' : ' at '
```

Comment:
[String.prototype.split can be used with a regex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split#Removing_spaces_from_a_string):
```suggestion
          const splitTag = / ((na|o)|(at)) /i;
```

Reply:
@rafaelverger Regex have a problem with the spaces on the split

Reply:
Yeah... split using regex must ignore matching groups, my bad. I've fixed the regex in the suggestion above from `/ ((na|o)|(at)) /i;` TO `/\s+(?:n[ao]|at)\s+/i;`

Here are the tests:
```js
const re = /\s+(?:n[ao]|at)\s+/i;
console.log('Job no Empresa'.split(re))          // [ 'Job', 'Empresa' ]
console.log('Job at Empresa'.split(re))          // [ 'Job', 'Empresa' ]
console.log('Job na Empresa'.split(re))          // [ 'Job', 'Empresa' ]
console.log('Job aT Empresa X'.split(re))        // [ 'Job', 'Empresa X' ]
console.log('Job NA Empresa X'.split(re))        // [ 'Job', 'Empresa X' ]
console.log('Job de Foo na Empresa X'.split(re)) // [ 'Job de Foo', 'Empresa X' ]
console.log('Job de Foo at Empresa X'.split(re)) // [ 'Job de Foo', 'Empresa X' ]
console.log('Job de Foo No Empresa X'.split(re)) // [ 'Job de Foo', 'Empresa X' ]
```